server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/index.ts

```diff
@@ -79,10 +86,23 @@ export function create(
 		collaborationSessionEventEmitter,
 	);
 
+	const throttlerConfig: IThrottlerConfig = {
+		tenantThrottlers,
+		restThrottleIdSuffix: Constants.alfredRestThrottleIdSuffix,
+		generalRestCallThrottleIdPrefix: Constants.generalRestCallThrottleIdPrefix,
+	};
+	const healthCheckEndpoints = createHealthCheckEndpoints(
+		"alfred",
+		readinessCheck,
+		false,
```

Comment:
nit: it can be helpful for readability to include the param name for primitive values like this. In this case, `false /* createLivenessEndpoint */,`